plugins/rect/README.md

```diff
@@ -0,0 +1,33 @@
+# rect
+
+This plugin adds rect (width + height) sizing utilities to Tailwind.
+
+## Usage
+
+```
+theme: {
+  rect: {
+    target: ['44px', '44px'],
+    '24': ['24px', '24px'],
+  },
+}
+variants: {
+  rect: ['responsive'],
+},
+plugins: [
+  require('@viget/tailwindcss-plugins/rect')(),
```

Comment:
@leobauza I revamped the `rect` plugin to accept config options. Lemme know what you think. Using the `rem()` function would happen up in the config instead of within the plugin, that way people can specify alternate units.

Reply:
Shoot I just found out you already rewrote this for FedPack at one point. My bad, I should have checked. Anyway, the non-configurable [UC version](https://github.com/vigetlabs/uchicago-microsite/blob/master/config/tailwind/plugins/rect.js) is what I started with, and here's [FedPack's version](https://github.com/vigetlabs/FedPack/blob/master/config/tailwind/plugins/rect.js).

Reply:
I liked how clean it was without specifying units, but if you think there's a use-case for alternative units, I can see a case for this

Reply:
Doing non-rem units might be a rare edge case. I'll streamline things towards the FedPack version, supporting unit-less numbers and rem output only. Then we can move this version to the examples directory in case somebody needs to support that edge case. 👍

Reply:
Awesome sounds good

Reply:
Rect updated and moved in 6175d6e.